javascript/0034-find-first-and-last-position-of-element-in-sorted-array.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Binary Search
+ * https://leetcode.com/problems/find-first-and-last-position-of-element-in-sorted-array/
+ * Time O(log(n)) | Space O(1)
+ * @param {number[]} nums
+ * @param {number} target
+ * @return {number[]}
+ */
+var searchRange = function(nums, target) {
+    
+    const result = [];
+
+    result.push(binarySearch(true));
+    result.push(binarySearch(false));
+
+    function binarySearch(leftBias) {
+      let left = 0;
+      let right = nums.length - 1;
+      let index = -1;
+
+      while(left <= right) {
+        
+        const mid = Math.floor((left+right)/2);
+
+        if(target > nums[mid]) {
+          left = mid+1;
+        }
+        if(target < nums[mid]) {
+          right = mid-1;
+        }
+        // this is the meat of the code
```

Comment:
Suggestion: Remove comment and creat descriptive variables or functions

```js
const isTarget = (...);
if (isTarget) { ... }
```

Reply:
done.